vos/vos/vos.py

```diff
@@ -1470,31 +1472,52 @@ def copy(self, source, destination, send_md5=False, disposition=False):
                     logging.debug("Got error {0}".format(ex))
                     continue
         else:
+            success = True
+            try:
+                destination_node = self.get_node(destination)
+                destination_md5 = destination_node.props.get('MD5', ZERO_MD5)
+            except Exception as ex:
+                destination_md5 = None
             source_md5 = md5_cache.MD5Cache.compute_md5(source)
-            put_urls = self.get_node_url(destination, 'PUT')
-            while not success:
-                if len(put_urls) == 0:
-                    if self.transfer_shortcut and not get_node_url_retried:
-                        put_urls = self.get_node_url(destination, method='PUT', full_negotiation=True)
-                        # remove the first one as we already tried that one.
-                        put_urls.pop(0)
-                        get_node_url_retried = True
-                    else:
-                        break
-                put_url = put_urls.pop(0)
-                try:
-                    with open(source, str('rb')) as fin:
-                        self.conn.session.put(put_url, data=fin)
-                    node = self.get_node(destination, limit=0, force=True)
-                    destination_md5 = node.props.get('MD5', ZERO_MD5)
-                    assert destination_md5 == source_md5
-                except Exception as ex:
-                    copy_failed_message = str(ex)
-                    logging.debug("FAILED to PUT to {0}".format(put_url))
-                    logging.debug("Got error: {0}".format(ex))
-                    continue
-                success = True
-                break
+            if source_md5 == destination_md5:
+                logger.info('copy: src and dest are already the same -> update node metadata')
```

Comment:
Can this be change to 'debug'.   Having as a verbose message will cause confusion when people call vcp which already makes a similar statement.   Or is this logger set to ERROR during normal operations?